mlflow/store/artifact/optimized_s3_artifact_repo.py

```diff
@@ -32,6 +32,22 @@
 _HTTP_HEADER_BUCKET_REGION = "x-amz-bucket-region"
 _BUCKET_LOCATION_NAME = "BucketLocationName"
 
+# allowed for complete_multipart_upload, upload_part
+# https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/complete_multipart_upload.html
+# https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/upload_part.html
+# https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/get_object.html
+_ALLOWED_EXTRA_ARGS = {
```

Comment:
Make variable public and import it in `s3_artifact_repo.py` to avoid duplicating code.
```suggestion
ALLOWED_MPU_EXTRA_ARGS = {
```